docs/user-guide/command-line-interface.md

```diff
@@ -81,6 +81,7 @@ Miscellaneous:
   --init                         Run config initialization wizard - default: false
   --env-info                     Output execution environment information - default: false
   --no-error-on-unmatched-pattern  Prevent errors when pattern is unmatched - default: false
+  --exit-on-fatal-error          Exit with exit code 2 in case of fatal error.  - default: false
```

Comment:
Can we figure out a different name? I think the term "fatal error" is misleading for this top-level option.

Being unable to parse code is a fatal error for `Linter`, but for the overall ESLint execution fatal errors are errors like malformed configuration files, missing plugins, bugs that cause uncaught exceptions etc., and in such cases ESLint exits with exit code 2 regardless of this option.

Since the longer description below specifically mentions parsing errors as those which this option targets, can we name the option `--exit-on-parsing-error`?

Reply:
We already label parsing errors as “fatal: true” and this was the name approved in the RFC, so we should discuss if a change is necessary at this stage.

Reply:
Both seem fine to me but I do get the point of @mdjermanovic that using the word `parsing` instead of `fatal` may make it more clear to the end user despite it not being really correlated with the source code.

Reply:
Okay, if `--exit-on-fatal-error` sounds good to everyone else, then it's better not to change it at this point.